index.html

```diff
@@ -11,25 +11,176 @@
+                            <li class="chips">css</li>
+                            <li class="chips">javascript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="work-button"><a class="project-button" href="#">See Project</a></div>
+            </div>
+        </div>
+        <div class="work-item">
+            <div class="work-box">
+                <div class="card-image">
+                    <img class="work-snapshot" src="images/SnapshotPortfolio2.png" alt="snapshot">
+                </div>
+                <h2 class="card-title">Tonic</h2>
+                <div class="work-detail-box">
+                    <ul>
+                        <li class=" work-detail canopy">CANOPY</li>
+                        <li class="work-detail work-text">Back End Dev</li>
+                        <li class="work-detail work-text">2015</li>
+                    </ul>
+                </div>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <div>
+                    <div class="work-detail-box">
+                        <ul>
+                            <li class="chips">html</li>
+                            <li class="chips">css</li>
+                            <li class="chips">javascript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="work-button"><a class="project-button" href="#">See Project</a></div>
+            </div>
+        </div>
+        <div class="work-item">
+            <div class="work-box">
+                <div class="card-image">
+                    <img class="work-snapshot" src="images/SnapshotPortfolio3.png" alt="snapshot">
+                </div>
+                <h2 class="card-title">Multi-Post <br> Stories</h2>
+                <div class="work-detail-box">
+                    <ul>
+                        <li class=" work-detail canopy">CANOPY</li>
+                        <li class="work-detail work-text">Back End Dev</li>
+                        <li class="work-detail work-text">2015</li>
+                    </ul>
+                </div>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <div>
+                    <div class="work-detail-box">
+                        <ul>
+                            <li class="chips">html</li>
+                            <li class="chips">css</li>
+                            <li class="chips">javascript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="work-button"><a class="project-button" href="#">See Project</a></div>
+            </div>
+        </div>
+
+        <div class="about-section">
```

Comment:
- [ ] I think it's better to use a `<section>` instead of div tag. It will make your webpage more semantic.